Branch-SDK/src/main/java/io/branch/referral/BranchShortLinkBuilder.java

```diff
@@ -137,10 +137,6 @@ public String getShortUrl() {
      * @param callback A {@link Branch.BranchLinkCreateListener} callback instance that will trigger
      */
     public void generateShortUrl(Branch.BranchLinkCreateListener callback) {
-        super.generateUrl(callback);
-    }
-
-    void generateShortUrlInternal(Branch.BranchLinkCreateListener callback, boolean isFromShareSheet) {
```

Comment:
I'm removing only internal methods that overload url generation with the argument `isFromShareSheet`, which apparently isn't used in any shape or form.